src/braket/pennylane_plugin/braket_device.py

```diff
@@ -643,3 +730,46 @@ def _run_task(self, circuit, inputs=None):
             inputs=inputs or {},
             **self._run_kwargs,
         )
+
+    def classical_shadow(self, obs, circuit):
+        if circuit is None:  # pragma: no cover
+            raise ValueError("Circuit must be provided when measuring classical shadows")
+
+        wires = obs.wires
+        n_snapshots = self.shots
+        seed = obs.seed
+
+        n_qubits = len(wires)
+        mapped_wires = np.array(self.map_wires(wires))
+        # seed the random measurement generation so that recipes
+        # are the same for different executions with the same seed
+        rng = np.random.default_rng(seed)
+        recipes = rng.integers(0, 3, size=(n_snapshots, n_qubits))
+        obs_list = [qml.PauliX, qml.PauliY, qml.PauliZ]
```

Comment:
Rather than instantiate this every time, can you extract this to a constant outside of the class? And make it a tuple so it's immutable:

```python
OBS_LIST = (qml.PauliX, qml.PauliY, qml.PauliZ)

class BraketQubitDevice(...):
    ...
```